chromium/debugging-rulesets.js

```diff
@@ -0,0 +1,20 @@
+/* global sendMessage */
+
+"use strict";
+
+window.onload = function() {
+  document.querySelector("form").addEventListener("submit", save_debugging_rulesets);
+
+  sendMessage("get_option", { debugging_rulesets: "" }, item => {
+    document.getElementById("debugging-rulesets").value = item.debugging_rulesets;
```

Comment:
This is a race condition in the UI. The `<textarea id="debugging-rulesets">` element will display before it is populated with the data.

This is a little bit jarring visually, but the big problem is that it makes things harder to test with selenium. Since selenium  might get the textarea's value before it is populated.

One way to fix this, is to add `style="display:none"` to the appropriate element in the HTML. Then in this script, change the style to `"block"` when you've populated the data. There are helpers for this in `popup.js` [here](https://github.com/EFForg/https-everywhere/blob/master/chromium/popup.js#L207-L213).

An example of fixes  like this I've made are [here](https://github.com/EFForg/https-everywhere/pull/12624/commits/74763c83120686e9e1b8513878a440a35cbf8f85).